doc/examples/widgets.lean

```diff
@@ -137,24 +137,21 @@ def checkWidget : UserWidgetDefinition where
   javascript := "
 import * as React from 'react';
 const e = React.createElement;
-import { RpcContext, InteractiveCode, useAsync, mapRpcError } from '@leanprover/infoview';
+import { RpcContext, InteractiveCode } from '@leanprover/infoview';
 
 export default function(props) {
   const rs = React.useContext(RpcContext)
   const [name, setName] = React.useState('getType')
+  const [value, setValue] = React.useState(undefined)
 
-  const [status, val, err] = useAsync(() =>
-    rs.call('getType', { name, pos: props.pos }), [name, rs, props.pos])
-
-  const type = status === 'fulfilled' ? val && e(InteractiveCode, {fmt: val})
-    : status === 'rejected' ? e('p', null, mapRpcError(err).message)
-      : e('p', null, 'Loading..')
+  function run() {
+    rs.call('getType', { name, pos: props.pos }).then(setValue)
```

Comment:
Why discourage the use of `useAsync`? It has better atomicity properties than a raw `rs.call`.

Reply:
Oh, I see we made a release of `vscode-lean4` which changed the signature of `useAsync`. I will make a new widget API release and reintroduce it here.

Reply:
Yes also `useAsync` wasn't available on the root ES module.